.ci/install_linux_cmake.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -ex
+
+$SUDO apt-get -qq update
+$SUDO apt-get -y install lsb-release software-properties-common
+if [ $(lsb_release -sc) = "trusty" ]; then
+  $SUDO apt-add-repository -y ppa:libccd-debs/ppa
+  $SUDO apt-add-repository -y ppa:fcl-debs/ppa
+fi
+$SUDO apt-add-repository -y ppa:dartsim/ppa
+$SUDO apt-get -qq update
+
+# Build tools
+$SUDO apt-get -y install \
+  sudo \
+  build-essential \
+  cmake \
+  pkg-config \
+  curl \
+  git
+if [ $COMPILER = clang ]; then
+  $SUDO apt-get -qq -y install clang
+fi
+
+# Required dependencies
+$SUDO apt-get -y install \
+  libboost-filesystem-dev \
+  libdart6-all-dev \
+  libompl-dev
+
+# Optional dependencies
+$SUDO apt-get -y install \
+  libtinyxml2-dev \
+  libyaml-cpp-dev
+
+if [ "$BUILD_AIKIDOPY" = "ON" ]; then
+  $SUDO apt-get -y install python3-dev python3-numpy
+  $SUDO apt-get -y install python3-pip -y
+  $SUDO pip3 install pytest -U
+
+  if [ $(lsb_release -sc) = "trusty" ]; then
```

Comment:
Is there a reason that the `trusty`, `xenial`, and `bionic` cases are duplicated? (And `cosmic` and `disco`.)

Reply:
Nope. Just for the case that they require different scripts. Feel free to merge them into two groups.